content/2020/2020-12/2020-12-11-pulsar-weekly.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "2020-12-11-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2020-12-05 ~ 2020-12-11"
+date: 2020-12-05 ~ 2020-12-11
+description: "This is the Pulsar community weekly update for 2020-12-05 ~ 2020-12-11, with updates on Pulsar client, broker, Functions, and so on."
+id: "2020-12-11-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-12-05 ~ 2020-12-11
+
+This is the Pulsar community weekly update for 2020-12-05 ~ 2020-12-11, with updates on Pulsar client, broker, Functions, and so on.
+
+### Pulsar Highlight
+
+- [Broker] Export Prometheus metrics for messageTTL.
+
+    https://github.com/apache/pulsar/pull/8871 
+
+    by ([@315157973](https://github.com/315157973))
+
+- [Function] Make the Schema information of the source topic available to downstream Sinks.
+
+    https://github.com/apache/pulsar/pull/8854 
+
+    by ([@eolivelli](https://github.com/eolivelli))
+
+- [Function] Support fetching URLs for Go and Python functions.
+
+    https://github.com/apache/pulsar/pull/8808 
+
+    by ([@wolfstudy](https://github.com/wolfstudy))
+
+- [Broker] Implement the packages management administration operations.
+
+    https://github.com/apache/pulsar/pull/8816 
+    
+    by ([@zymap](https://github.com/zymap))
+
+### Notable Feature
+
+- [Broker] Export Prometheus metrics for messageTTL.
+
+    https://github.com/apache/pulsar/pull/8871 ([@315157973](https://github.com/315157973))
+    
+- [Broker] Add REST APIs for package management service.
+
+    https://github.com/apache/pulsar/pull/8858 ([@zymap](https://github.com/zymap))
+    
+- [Broker] Add the package management service into the Pulsar startup process.
+
+    https://github.com/apache/pulsar/pull/8764 ([@zymap](https://github.com/zymap))
+
+- [Broker] Change method `getWebServiceUrl` into async.
+
+    https://github.com/apache/pulsar/pull/8746 ([@reswqa](https://github.com/reswqa))
+    
+- [Broker] Support the package management client command.
+
+    https://github.com/apache/pulsar/pull/8817 ([@zymap](https://github.com/zymap))
+    
+- [Broker] Implement the packages management administration operations.
+
+    https://github.com/apache/pulsar/pull/8816 ([@zymap](https://github.com/zymap))
+    
+- [Function] Make Source topic Schema information available to downstream Sinks.
+
+    https://github.com/apache/pulsar/pull/8854 ([@eolivelli](https://github.com/eolivelli))
+
+- [Function][Function Worker] Splitting the authentication logic of function worker and client.
+
+    https://github.com/apache/pulsar/pull/8824 ([@nodece](https://github.com/nodece))
+    
+- [Function] Support fetching URLs for Go and Python functions.
+
+    https://github.com/apache/pulsar/pull/8808 ([@wolfstudy](https://github.com/wolfstudy))
+
+- [Function] Add the command flag of `--retain-key-ordering` to Pulsar Functions .
```

Comment:
1. Is `--retain-key-ordering` a command flag?
2. remove the redundant space at the end.

Reply:
> 1. Is `--retain-key-ordering` a command flag?

--[Huanli]: according to the PR description, it is a cmd flag.
> 2. remove the redundant space at the end.

-- Huanli: removed.